pydeconz/models/light/light.py

```diff
@@ -215,9 +236,12 @@ class LightEffect(enum.StrEnum):
 
     CANDLE = "candle"
     FIREPLACE = "fireplace"
-    LOOP = "loop"
```

Comment:
Why did you remove loop?

Reply:
`loop` has been renamed `prism` but it shouldn't have been removed - thanks for catching this.